dsl/src/main/java/tech/pegasys/peeps/network/ConsensusMechanism.java

```diff
@@ -17,5 +17,6 @@ public enum ConsensusMechanism {
   IBFT2,
   CLIQUE,
   QBFT,
+  QBFT_SMART_CONTRACT,
```

Comment:
This is a bit of hack defining a new type. It's not really another consensus mechanism just a configuration of QBFT.